lerobot/scripts/visualize_image_transforms.py

```diff
@@ -0,0 +1,46 @@
+from pathlib import Path
+
+import hydra
+from torchvision.transforms import ToPILImage
+
+from lerobot.common.datasets.lerobot_dataset import LeRobotDataset
+from lerobot.common.datasets.transforms import make_image_transforms
+
+to_pil = ToPILImage()
+
+
+def main(cfg, output_dir=Path("outputs/image_transforms")):
```

Comment:
Could you provide a docstring in header of the script + example command?

Reply:
Fixed https://github.com/huggingface/lerobot/pull/234/commits/1fbb0d9467b6ac4435103d60be44e234fda8a784